### ISSUE TITLE
U+21AE LEFT RIGHT ARROW WITH STROKE has strange stroke placement

### ISSUE THREAD

**Issue by dankamongmen** [silnrsi/font-gentium#29]
Hello again! I've been using U+21AE LEFT RIGHT ARROW WITH STROKE (↮), and Gentium has made the strange IMHO choice of placing the stroke on the far right of the glyph.

It's not the only font that does this -- I see similar behavior from FreeSerif (though only in italic), Cousine, Charis SIL (all forms), and others. Nonetheless, this seems incorrect. Proper glyph formation, with the stroke in the center, can be seen in Asana Math, DejaVu, Linux Libertine, JetBrains Mono, and others.

Indeed, in my browser, https://www.compart.com/en/unicode/U+21AE renders with the stroke far off to the right, but it gets https://www.compart.com/en/unicode/U+21CE correct:

↮
 ⇎

so this clearly isn't a problem with just Gentium, but i do think it's a definite bug. left right arrow with stroke is clearly intended as a foil to left right arrow. the latter is a symbol for tautology (https://en.wikipedia.org/wiki/Tautology_(logic)) whereas the former is a symbol for exclusive-or (https://en.wikipedia.org/wiki/Exclusive_or).

perhaps you know something i don't, of course. otherwise, it would be great if the stroke could be in the middle. actually, i see that this affects U+21CE LEFT RIGHT DOUBLE ARROW WITH STROKE in gentium also.

if this needs be fixed in some Ur-font from which gentium inherits, let me know, and i'll go report a bug on them, too. 

(btw i noticed that my Debian machine only has 6.102; i filed a bug to upgrade the debian package to 6.200)

**Comment by dankamongmen** [comment]
i'm looking forward to using the new logical symbols from gentium 6.2, btw! this table will look a lot better

![2023-09-05-162250_511x142_scrot](https://github.com/silnrsi/font-gentium/assets/143473/15cb846c-2096-415e-b918-70962ed19a53)

**Comment by dankamongmen** [comment]
it sure does!

![2023-09-05-180916_502x160_scrot](https://github.com/silnrsi/font-gentium/assets/143473/ec07e13e-f995-4f13-8c9e-d069c842d365)

it would be great to get U+22BD NOR and U+22BC NAND to go along with OR and AND! ⊽⊼

**Comment by dankamongmen** [comment]
ooooh, i was able to fake U+22BD NOR and U+22BC NAND with $\overbar{\mbox{X}}$, so now my table is perfect (presentation-wise anyway) except for U+21AE.

**Comment by jvgaultney** [comment]
Thanks!

**Comment by jvgaultney** [comment]
Hmmm. If you need both U+21AE and the vertical stroke counterpart (U+21F9) then wouldn't you also need U+21F7 and U+21F8? Do you really need both U+21AE and U+21F9?

**Comment by dankamongmen** [comment]
> Hmmm. If you need both U+21AE and the vertical stroke counterpart (U+21F9) then wouldn't you also need U+21F7 and U+21F8? Do you really need both U+21AE and U+21F9?

nope, i only need one! i was just saying that if 21F9 was available, it would also suffice. i agree that 21ae makes more sense given the glyphs you're already supplying.

**Comment by jvgaultney** [comment]
Got it thanks

**Comment by dankamongmen** [comment]
awesome!
